content/en/blog/2020/multiple-control-planes/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Multiple Control Planes"
```

Comment:
@howardjohn - may I suggest the title "Immutable Control Planes". This is groundbreaking work and should not be confused with multicluster and the pile of terms we have in there.

Plus @oaktowner will have something good to work with during the release. This may require a little rework on this document.

Reply:
I don't have any strong opinions on what we call this. It might make sense as "Control Plane Revisions" since that is how it is configured. Or for a title something like "Immutable Control Planes: introducing revisions" or something like that. Naming things is not my expertise, so I'll defer to others :slightly_smiling_face:

Reply:
Well, given that the blog is talking about canary deployment of a control plane, the title should probably be something like:
```
Safely Upgrade Istio using a Canary Control Plane Deployment
```